docs/source/pages/using.rst

```diff
@@ -205,3 +207,35 @@ Thanks to Datalog’s expressive language model, it’s easy to add exception ru
 requirement. For example, `the Mysql Connector/J <https://slsa.dev/spec/v0.1/requirements#build-service>`_ dependency in
 the Micronaut MuShop project does not pass the ``build_service`` check, but can be manually investigated and exempted if trusted. Overall, policies expressed in Datalog can be
 enforced by Macaron as part of your CI/CD pipeline to detect regressions or unexpected behavior.
+
+-----------------------------
+Analyzing a GitLab repository
+-----------------------------
```

Comment:
Adding the docs for GitLab after running the policy engine breaks the flow. How about adding a top level section for analyzing different types of version control systems, and adding GitHub, GitLab, and locally cloned repos as its subsections? The section headline could be:
``` 
Collecting evidence from the source code repository
```

Reply:
Fixed in f6d72ed8b1e0dd37e2571812f95913089d61e6f4.